### PR TITLE
 Documentation on standalone installer and vtag for csi unity driver

### PR DESCRIPTION
**Pull request by karthikk92**
# Description
Documentation on standalone installer and vtag for csi unity driver
# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/243 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
standalone helm install : 

sanity runs for nightly and v2.2.0 build :

### CODE REVIEW

File: helm/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+The general outline is:
+
+    1. Satisfy the pre-requsites outlined in the Release and Installation Notes in the doc directory.
+
+    2. Create a Kubernetes secret with the Unity credentials using the template in secret.yaml.
+
+    3. Make a copy of the `csi-unity/values.yaml` to the location of your choice(say csi-unity/myvalues.yaml) and fill in various installation parameters.
+
+    4. Run the helm install command, first using the --dry-run flag to confirm various parameters are as desired.
+       Once the parameteres are validated, run the command without the --dry-run flag.
```
Comment (rensyct) [inline helm/README.md:14]: Spell check 
parameteres -> parameters

File: helm/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+       Note: The below example assumes that the user is at repo root helm folder i.e csi-unity/helm.
+              
+       Syntax: helm install --dry-run --values <myvalues.yaml location> --namespace <namespace> <name of secret> <helmPath>
+       <name of secret> - unity in case of unity-creds and unity-certs-0 secrets.
```
Comment (rensyct) [inline helm/README.md:18]: Rearrange the order of the parameters as provided in the syntax

Comment (karthikk92) [reply helm/README.md:19]: taken care

File: helm/README.md (hunk 1 above)
Comment (karthikk92) [reply helm/README.md]: taken care

**Review by rensyct — APPROVED** [review]

**Review by rajendraindukuri — APPROVED** [review]

**Review by panigs7 — APPROVED** [review]

**Pull request merged**